### ISSUE TITLE
Ease curve always starts at 0,0

### ISSUE THREAD

**Issue by swingingtom** [Nowan/dat.gui.ease#3]
When using a custom ease that doesn't start at `0,0` point, the `0,0` point is forced anyway.

consider this curve : 
`CustomEase.create("custom", "M0,0.544 C0.134,0.748 0.204,0.806 0.434,0.806 0.697,0.806 1,0.306 1,0 ");`
https://greensock.com/docs/v3/Eases?CustomEase=M0,0.544%20C0.134,0.748%200.204,0.806%200.434,0.806%200.697,0.806%201,0.306%201,0

![image](https://user-images.githubusercontent.com/17314210/143764932-5faac6e1-4fcf-4c6d-993a-f4c74eec6927.png)

once added in gui it becomes : 
```javascript
let myEase = new CustomEase("clampedAt00","M0,0.544 C0.134,0.748 0.204,0.806 0.434,0.806 0.697,0.806 1,0.306 1,0 ");
gui.addEase({ ease:  myEase }, "ease");
```
![image](https://user-images.githubusercontent.com/17314210/143764838-74d65982-adf2-491a-86ec-79be18c2cf00.png)

No error nor warnings in the console.

**Comment by swingingtom** [comment]
fix #4

**Comment by Nowan** [comment]
#4 merged
